client.py

```diff
@@ -0,0 +1,94 @@
+"""
+Implement a tkinter client who communicates with a fastapi server
+and executes commands
+"""
+
+import requests
+from threading import Thread
+import tkinter
+
+
+root = tkinter.Tk()
+frame = tkinter.Frame(root)
+command_inp = tkinter.Text(frame, width=70, height=1, fg="green")
```

Comment:
Why are these global variables? Why aren't they passed as parameters and defined inside of the `main` function or something?